Sources/Plasma/NucleusLib/pnNetCli/pnNcCli.cpp

```diff
@@ -356,11 +366,18 @@ static void BufferedSendData (
                 // we reserve one space for the NULL terminator
                 const uint16_t length = (uint16_t) std::char_traits<char16_t>::length((const char16_t *) *msg);
                 ASSERT_MSG_VALID(length < cmd->count);
+
                 // Write actual string length
                 uint16_t size = hsToLE16(length);
                 AddToSendBuffer(cli, sizeof(uint16_t), (const void*)&size);
+
+                std::vector<char16_t> temp(length);
+                for (size_t i = 0; i < length; i++) {
+                    temp[i] = hsToLE16(((char16_t*)*msg)[i]);
+                }
+
                 // Write string data
-                AddToSendBuffer(cli, length * sizeof(char16_t), (const void *) *msg);
+                AddToSendBuffer(cli, length * sizeof(char16_t), (const void*)temp.data());
```

Comment:
Do we need to explicitly cast this to `const void*`?

Reply:
No idea, I just kept the cast that it had previously 😛